ui-tests/base/com.liferay.ide.functional.kaleo.deploy.base/src/com/liferay/ide/functional/kaleo/deploy/base/DeployKaleoGradleTomcat7xBase.java

```diff
@@ -38,7 +38,7 @@
 	public void deployKaleoWorkflowAssignCreatorOnProject() {
 		wizardAction.openNewLiferayModuleWizard();
 
-		wizardAction.newModule.prepare(project.getName());
+		wizardAction.newModule.prepareGradle(project.getName(), MVC_PORTLET, "7.0");
```

Comment:
we already have a method prepareGradle for creating 70 module in wizardAction.java, 

> public void prepareGradle(String projectName, String template) {
> 			_prepare(projectName, GRADLE, template, "7.0");
> 		}

please update to:
prepareGradle(project.getName(),MVC_PORTLET);